DEVELOPING.md

```diff
@@ -22,8 +22,12 @@ We discuss implementation issues on [amphtml-discuss@googlegroups.com](https://g
 
 ### Installation
 
-1. `npm i`
-2. `edit /etc/hosts` and map `ads.localhost` and `iframe.localhost` to `127.0.0.1`:
+1. Install NodeJS from https://nodejs.org/
```

Comment:
There are other installation mechanisms (e.g. brew) - let's not prefer one.  Just say "Install NodeJS" (you can link NodeJS to nodejs.org).